index.js

```diff
@@ -38,11 +38,12 @@ nunjucks.configure(app.get('views'), {
 });
 
```

Comment:
```
const renderComments() => {
  return nunjucks.render('comments.njk', { comments: db.get() });
};

app.get('/', (request, response) => {
  const options = {
  ...
  commentsHTML: renderComments()
  };
  return response.render('home', options);
};
```

Reply:
Yes, that is nicer.
const renderComments = () => {
...stuff
}